test/evaluation_log_test.exs

```diff
@@ -0,0 +1,141 @@
+defmodule ConfigCat.EvaluationLogTest do
+  # We change the logging level for these tests, so we run them synchronously.
+  use ConfigCat.Case, async: false
+
+  import ExUnit.CaptureLog
+
+  alias ConfigCat.CachePolicy
+  alias ConfigCat.LocalFileDataSource
+  alias ConfigCat.NullDataSource
+  alias ConfigCat.User
+
+  @moduletag capture_log: true
+
+  setup do
+    original_level = Logger.level()
+    Logger.configure(level: :debug)
+    on_exit(fn -> Logger.configure(level: original_level) end)
+  end
+
+  test "simple value" do
+    test_evaluation_log("simple_value.json")
+  end
+
+  test "1 targeting rule" do
+    test_evaluation_log("1_targeting_rule.json")
+  end
+
+  test "2 targeting rules" do
+    test_evaluation_log("2_targeting_rules.json")
+  end
+
+  test "options based on user id" do
+    test_evaluation_log("options_based_on_user_id.json")
+  end
+
+  test "options based on custom attr" do
+    test_evaluation_log("options_based_on_custom_attr.json")
+  end
+
+  test "options after targeting rule" do
+    test_evaluation_log("options_after_targeting_rule.json")
+  end
+
+  test "options within targeting rule" do
+    test_evaluation_log("options_within_targeting_rule.json")
+  end
+
+  test "and rules" do
+    test_evaluation_log("and_rules.json")
+  end
+
+  test "segment" do
+    test_evaluation_log("segment.json")
+  end
+
+  test "prerequisite flag" do
+    test_evaluation_log("prerequisite_flag.json")
+  end
+
+  test "semver validation" do
+    test_evaluation_log("semver_validation.json")
+  end
+
+  test "epoch date validation" do
+    test_evaluation_log("epoch_date_validation.json")
+  end
+
+  test "number validation" do
+    test_evaluation_log("number_validation.json")
+  end
+
+  test "comparators validation" do
+    # self.maxDiff = None
```

Comment:
Is this line needed here?

Reply:
Nope, left over from when I copied in the python code. I wasn't sure if I'd need something like it and then forgot to delete it.